changelog.md

```diff
@@ -0,0 +1,22 @@
+**Beta features**
+1. Multiple sidechain nodes network.
+2. Mainchain synchronization: Cross-chain Transfer Protocol support for sidechain declaration: forward and backward transfers.
+3. Proof-of-Stake consensus protocol for sidechain based on Ouroboros Praos consensus protocol.
+4. Built-in coins transferring operations inside sidechain.
```

Comment:
Let's specify the new kind of transaction - SidechainCoreTranscation and also spesify the Boxes structure was improved/changed.